### PR TITLE
Splitting up the view repositories

### PR DESCRIPTION
**Pull request by mamazu**
Moving the view repositories in their own namespaces.

### CODE REVIEW

File: src/Resources/config/services/controllers.xml [hunk 1]
```diff
@@ -2,12 +2,6 @@
...
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <imports>
-        <import resource="actions/address_book.xml"/>
-        <import resource="actions/checkout.xml"/>
-        <import resource="actions/cart.xml"/>
-        <import resource="actions/customer.xml"/>
-        <import resource="actions/order.xml"/>
-        <import resource="actions/product.xml"/>
-        <import resource="actions/taxon.xml"/>
+        <import resource="actions/**/*"/>
```
Comment (lchrusciel) [inline src/Resources/config/services/controllers.xml]: Can't we just use generic import from services.xml?

Comment (mamazu) [reply src/Resources/config/services/controllers.xml]: Sure, I'll try to do that. But some files aren't split up properly.

Comment (lchrusciel) [reply src/Resources/config/services/controllers.xml]: That is ok. This PR may be merged, but first I need to make my mind with #351.

**Comment by lchrusciel** [comment]
Thank you, @mamazu! :1st_place_medal:

**Pull request merged**